style.css

```diff
@@ -497,7 +498,7 @@ html {
     height: 718px;
     background-color: #fff;
     background-repeat: no-repeat;
-    margin-top: 72px;
+    margin-top: 50px;
     display: flex;
     justify-content: center;
     align-items: start;
```

Comment:
- [ ] Great work :muscle: but I think you have a few things to match the project requirements as requested that when a user clicks on any of the navigation link, it should navigate them to the corresponding sections. After checking, I saw that it was probably due to the wrong `id` attributes that you have in your `a href="#id"` because it is not the same with what you have in the `section` tags - `section id="id"`.
- [ ] [OPTIONAL] In a bid to increase accessiblity and user experience, I recommend that you introduce a `cursor: pointer` to your `a` tags in the `css` file.

Reply:
Fixed.